staging/gatekeeper/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "3.2.2"
```

Comment:
shouldn't this change....? `3.3.0` perhaps?

Reply:
yeah at this point;